Dockerfile.runtime.template

```diff
@@ -0,0 +1,41 @@
+# Minimal runtime image
+FROM resin/%%ARCH%%-supervisor-base
```

Comment:
Ok, that explains why I couldn't find it, although I'm still not sure precisely what's in it (I have a feeling a high level of yocto is needed to know the full details) but at least I know the idea behind it is the minimal busybox + nodejs :)